ddtrace/mocktracer/mockspan.go

```diff
@@ -284,3 +291,10 @@ func (s *mockspan) Root() tracer.Span {
 	root, _ := current.(*mockspan)
 	return root
 }
+
+// Source returns the component from which the mockspan was created.
+// This is used to test the source tag of the `datadog.tracer.spans_{started,finished}`
+// health metrics.
```

Comment:
```suggestion
// Integration returns the component (integration) from which the mockspan was created.
```
I'd probably resist tying this method to a specific test as it's possible it could be used for other tests in the future.